app/helpers/firm_helper.rb

```diff
@@ -24,4 +24,12 @@ def minimum_pot_size_text(firm_result)
   def minimum_fixed_fee(firm_result)
     number_to_currency firm_result.minimum_fixed_fee, unit: '£', precision: 0
   end
+
+  def link_with_protocol(link)
+    if link =~ /^https?/
+      link
+    else
+      "http://#{link}"
+    end
+  end
```

Comment:
Yep, so you probably shouldn't use it in that case. Maybe it could be named better? `ensure_external_link_has_protocol`?